### DISCUSSION TITLE
How to test replicated tables?

### DISCUSSION THREAD

**Discussion by dozer111** [ClickHouse/ClickHouse#59952] [Q&A]
I`m sure this is common problem. There is integration test with read db. 

For example, i had table with "insert_quorum = 2"

So, the simplest solution is create 3 ch nodes, create zookeeper, connect them. Not so simple at all(

Maybe, there is some settings/approaches, or sth else, which make the test easy

**Comment by Unalian** [comment]
Maybe you can read tests/integration/readme.md.
